core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -36,34 +33,22 @@ <h2 class="activity-title"><[getCollectionTitle()]></h2>
           <div class="title-section-mask top-card-mask"></div>
         </a>
       </md-card>
-      <div class="title-section-mask"></div>
-    </md-card>
-    <div class="title-section-mask"></div>
-  </md-card>
 </script>
 
 <style>
```

Comment:
See if the existing CSS attributes for the `oppia-activity-summary-tile` works for this new collections tile (if so, remove the parts that deal with the card itself and keep the corner image attributes), that'll help make the code dry.